zebedee-cms/src/main/java/com/github/onsdigital/zebedee/session/store/SessionsThreadLocalImpl.java

```diff
@@ -0,0 +1,54 @@
+package com.github.onsdigital.zebedee.session.store;
+
+import com.github.onsdigital.exceptions.*;
+
+import com.github.onsdigital.exceptions.SessionsDecodeException;
+import com.github.onsdigital.exceptions.SessionsTokenExpiredException;
+import com.github.onsdigital.exceptions.SessionsVerificationException;
+import com.github.onsdigital.impl.*;
+import com.github.onsdigital.interfaces.*;
+import javax.servlet.http.HttpServletRequest;
+
+public class SessionsThreadLocalImpl implements SessionsThreadLocal {
+
+    private static HttpServletRequest request;
+    private static String secretKey;
+    private static ThreadLocal<UserDataPayload> store =
+            new ThreadLocal<>();
+
+    private JWTHandler jwtHandler;
```

Comment:
You're going to have to initialise the `jwtHandler` object, by doing something like:

`private JWTHandler jwtHandler = new JWTHandlerImpl();`